### ISSUE TITLE
Send fleet in expedition lost in deep space

### ISSUE THREAD

**Issue by BigTwoProduction** [jkroepke/2Moons#297]
Hello,

Since some time fleet does not return in "Destruction Mission", fleet is lost , no return, the fleet is block in admin panel, in log i see this error:
`[28-May-2017 15:01:27] USER ERROR: "SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'lostunits    = lostunits + '55000',
                 desunits        = desunits + '18188000'
                 WHERE id I' at line 5^M
 ^M
 Query-Code:UPDATE uni1_users SET
                 `wons` = `wons` + 1,
                 kbmetal         = kbmetal + 3044100,
                 kbcrystal       = kbcrystal + 2428800,K
                 lostunits       = lostunits + 55000,
                 desunits        = desunits + 18188000
                 WHERE id IN (360);"^M
 File: /var/www/html/xnovareborn/includes/classes/Database.class.php | Line: 109^M
 URL: https://www.big-two.tk/xnovareborn/game.php?page=overview | Version: 1.8.git^M
 Stack trace:^M
 #0 /includes/classes/Database.class.php(160): Database-&gt;_query('UPDATE uni1_use...', Array, 'update')
 #1 /includes/classes/missions/MissionCaseDestruction.class.php(598): Database-&gt;update('UPDATE %%USERS%...', Array)
 #2 /includes/classes/class.FlyingFleetHandler.php(80): MissionCaseDestruction-&gt;TargetEvent()
 #3 /includes/FleetHandler.php(31): FlyingFleetHandler-&gt;run()
 #4 /includes/common.php(137): require('/var/www/html/x...')
 #5 /game.php(24): require('/var/www/html/x...')
 #6 {main}^`
and when I unblock them in admin panel, when i return to the game I have is this error.

`[28-May-2017 13:54:27] NOTICE: "Undefined index: universe"^M
File: /var/www/html/xnovareborn/includes/classes/class.PlanetRessUpdate.php | Line: 96^M
URL: https://www.big-two.tk/xnovareborn/game.php | Version: 1.8.git^M
Stack trace:^M
#0 /includes/classes/class.PlanetRessUpdate.php(96): errorHandler(8, 'Undefined index...', '/var/www/html/x...', 96, Array)
#1 /includes/classes/missions/MissionCaseDestruction.class.php(84): ResourceUpdate-&gt;CalcResource(Array, false, true, '1490109968')
#2 /includes/classes/class.FlyingFleetHandler.php(80): MissionCaseDestruction-&gt;TargetEvent()
#3 /includes/FleetHandler.php(31): FlyingFleetHandler-&gt;run()
#4 /includes/common.php(137): require('/var/www/html/x...')
#5 /game.php(24): require('/var/www/html/x...')
#6 {main}^M
`
Did you know why that happens ?

Best regards,
BigTwoProd.

**Comment by jkroepke** [comment]
Duplicate of https://github.com/jkroepke/2Moons/issues/296
